### PR TITLE
admin can destroy and see list of user and search user

### PR DESCRIPTION
**Pull request by ggbond2605**

### CODE REVIEW

File: app/views/admin/users/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<div class="page-header">
+  <h1><%= t "admin.user.list_users" %></h1>
+</div>
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <div id="search_form" class="row">
+      <%= search_form_for [:admin, @search], class: "form-inline" do |f| %>
+    <div class="form-group">
```
Comment (hellovietnam93) [inline app/views/admin/users/index.html.erb]: ident

File: app/views/admin/users/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+          <%= f.label :username_cont, t("admin.user.username") %>
+          <%= f.text_field :username_cont, class: "form-control" %><br>
+        </div>
+        <%= f.submit t("admin.user.search"), class: "btn btn-info" %>
+      <% end %>
+    </div>
+  </div>
+</div>
+<div>
+<%= link_to t(:create_user), new_admin_user_path, class: "btn btn-success" %>
```
Comment (NgocVu1994) [inline app/views/admin/users/index.html.erb]: thut dong

**Comment by NeverSmileK57CLC** [comment]
OK

**Comment by hellovietnam93** [comment]
OK

**Pull request merged**